WORKSPACE

```diff
@@ -6,9 +6,7 @@ load("@io_bazel_rules_go//go:deps.bzl", "go_register_toolchains", "go_rules_depe
 
 go_rules_dependencies()
 
-go_register_toolchains(
-    version = "1.19.4",
-)
+go_register_toolchains(version = "1.19.4")
```

Comment:
This is back to 1.19.4. Given that 1.20 isn't out yet, maybe we should test it in `go_bazel_test` instead?

Reply:
Added a simple test doing a build in 1.20.